system-tests/Systemtests

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+pushd `dirname $0` > /dev/null
+SCRIPTPATH=`pwd`
```

Comment:
Actually, I tried this and it's not working. The pushd/popd approach did work though. 
But anyway, there is no need for this since the script is always running with `cwd` pointing to it's current location, so we can use relative paths